src/redemptions/redemption.service.ts

```diff
@@ -93,7 +93,7 @@ export class RedemptionService {
     }
 
     return {
-      status: KycStatus.TRY_AGAIN,
+      status: KycStatus.SUBMITTED,
```

Comment:
This is not a blocking comment but this is a dangerous change if you haven't carefully checked every case of your above checks. It's fine but if we have bugs users will pass KYC when they should fail. That's the main issue here.

I would generally assume all failures and whitelist specific cases rather than black list failure it's much safer.

Reply:
yes, this is the approach we chose. 

The more specific solution here will be https://airtable.com/appIXmGgVqP9QdbCf/tblAWSTOQbTq6vh6X/viwfithULCYLCL6B6/recAsHBMu6gviOgZq?blocks=hide